tests/js/spec/views/settings/auditLogView.spec.jsx

```diff
@@ -15,27 +22,40 @@ describe('OrganizationAuditLog', function () {
     });
   });
 
-  it('renders', async function () {
-    const wrapper = mountWithTheme(
-      <OrganizationAuditLog location={{query: ''}} params={{orgId: org.slug}} />
-    );
-    wrapper.setState({loading: false});
-    wrapper.update();
-    await tick();
+  it('renders', async () => {
+    render(<OrganizationAuditLog location={{query: ''}} params={{orgId: org.slug}} />, {
+      context: routerContext,
+    });
 
-    wrapper.update();
-    expect(wrapper).toSnapshot();
+    expect(await screen.findByRole('heading')).toHaveTextContent('Audit Log');
+    expect(screen.getByRole('textbox')).toBeInTheDocument();
+    expect(screen.getByText('Member')).toBeInTheDocument();
+    expect(screen.getByText('Action')).toBeInTheDocument();
+    expect(screen.getByText('IP')).toBeInTheDocument();
+    expect(screen.getByText('Time')).toBeInTheDocument();
+    expect(screen.queryByText('No audit entries available')).not.toBeInTheDocument();
```

Comment:
Is there a fragment of text in the audit log we could look for? `.not` on text queries can easily pass when they should fail. For example if the text had a typo in 'available' this assertion would continue to pass even though the 'empty' state is being shown.